front/src/applications/operationalStudies/hooks/useScenarioData.ts

```diff
@@ -110,14 +111,16 @@ const useScenarioData = (
   );
 
   // TODO Paced train : Adapt this to accept paced trains in issue https://github.com/OpenRailAssociation/osrd/issues/10613
-  useAutoUpdateProjection(infra, timetable.train_ids, trainScheduleSummaries);
+  useAutoUpdateProjection(
+    infra,
+    trainSchedulesResults.map((t) => t.id),
+    trainScheduleSummaries
+  );
 
   useEffect(() => {
     const fetchTrains = async () => {
-      const rawTrainSchedules = await fetchTrainSchedules({
-        body: {
-          ids: timetable.train_ids,
-        },
+      const rawTrainSchedules = await fetchAllTimetableByIdTrainSchedules({
```

Comment:
This hook now takes the list of train schedules as input. Here we're fetching the train schedules again.

There are two ways to fix this: either fetch the train schedules once in `useScenario()`, either fetch them once in `useScenarioData()`. I have a small preference for fetching them in `useScenarioData()` personally.

To fetch train schedules in `useScenarioData()`, we can drop `timetable` from the values returned by `useScenario()`, remove `trainSchedulesResults` from the props here, and then use `trainSchedules` instead of `trainSchedulesResults`.